scipy/spatial/qhull_misc_config.h

```diff
@@ -0,0 +1 @@
+#define HAVE_OPEN_MEMSTREAM 1
```

Comment:
I don't think you wanted to commit this file.

Reply:
I didn't